lms/djangoapps/courseware/grades.py

```diff
@@ -740,19 +736,6 @@ def get_score(user, problem_descriptor, module_creator, scores_client, submissio
     return weighted_score(correct, total, problem_descriptor.weight)
 
 
-@contextmanager
-def manual_transaction():
-    """A context manager for managing manual transactions"""
-    try:
-        yield
-    except Exception:
-        transaction.rollback()
-        log.exception('Due to an error, this transaction has been rolled back')
-        raise
-    else:
-        transaction.commit()
-
```

Comment:
The only difference between this and Django's `commit_on_success` was that this had logging. Lets just use the version of `commit_on_success` from `utils/db.py` instead.